templates/base.html.twig

```diff
@@ -99,7 +99,9 @@
             </div>
         {% endfor %}
 
-        {% block banner include('banner/call.html.twig', { banner: get_home_block(constant('AppBundle\\Entity\\HomeBlock::HEADER_BANNER')) }) %}
+        {% block banner %}
+            {{ include('banner/call.html.twig', { banner: get_home_block(constant('AppBundle\\Entity\\HomeBlock::HEADER_BANNER')) }) }}
```

Comment:
Is this always needed ? We said that we will not authorize connection without a verified email no ?